frontend/src/components/documentation/Routes/Analytics/UsersPerAppPerDept.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import Cell from './../../Cell.jsx'
+import Table from './../../Table.jsx'
+import Topic from './../../Topic.jsx'
+
+const codeExamples = {
+  python: `import requests
+
+params = {
+  "token": "uclapi-5d58c3c4e6bf9c-c2910ad3b6e054-7ef60f44f1c14f-a05147bfd17fdb",
+}
+r = requests.get("https://uclapi.com/dashboard/api/analytics/oauth/total_by_dept/", params=params)
+print(r.json())`,
+
+  shell: `curl -G https://uclapi.com/dashboard/api/analytics/oauth/total_by_dept/ \
+-d token=uclapi-5d58c3c4e6bf9c-c2910ad3b6e054-7ef60f44f1c14f-a05147bfd17fdb`,
+
+  javascript: `fetch("https://uclapi.com/dashboard/api/analytics/oauth/total_by_dept/?token=
+  uclapi-5d58c3c4e6bf9c-c2910ad3b6e054-7ef60f44f1c14f-a05147bfd17fdb")
+.then((response) => {
+  return response.json()
+})
+.then((json) => {
+  console.log(json);
+})`,
+}
+
+const response = `{
+  "ok": true,
+  "data": [
+    {
+      "department": "Dept of Computer Science",
+      "count": 100,
+    },
+    {
+      "department": "Dept of Mechanical Engineering",
+      "count" 50,
+    }
+  ]
+}
+`
+
+const responseCodeExample = {
+    python: response,
+    javascript: response,
+    shell: response,
+}
+
+export default class UsersPerAppPerDept extends React.Component {
+  render() {
+    const { activeLanguage } = this.props
+    return (
+      <div>
+        <Topic
+          activeLanguage={activeLanguage}
+          codeExamples={codeExamples}
+        >
+          <h1 id="dashboard/api/analytics/oauth/total">Total Number of Users for an App by Department</h1>
```

Comment:
This should probably be `dashboard/api/analytics/oauth/total_by_dept` since that's what it is at line 224 of `frontend/src/components/documentation/Sidebar.jsx`.